superset/connectors/sqla/models.py

```diff
@@ -698,6 +703,7 @@ def data(self) -> Dict[str, Any]:
             data_["fetch_values_predicate"] = self.fetch_values_predicate
             data_["template_params"] = self.template_params
             data_["is_sqllab_view"] = self.is_sqllab_view
+            data_["health_check_message"] = self.health_check_message
```

Comment:
instead of output whole "extra", i saw certified data only offer flattened information.